src/client/components/SessionTimer/index.js

```diff
@@ -16,11 +16,23 @@ const TIMEOUT_KEY = "timeout";
 function SessionTimer(props) {
   const { t } = useTranslation();
   const TIMEOUT_MS = 30 * 60 * 1000;
-  const TIMEOUT_WARNING_MS = TIMEOUT_MS - 5 * 60 * 1000;
+  const TIMEOUT_DISPLAY_TIME_IN_MINUTES = 5;
+  const TIMEOUT_WARNING_MS =
+    TIMEOUT_MS - TIMEOUT_DISPLAY_TIME_IN_MINUTES * 60 * 1000;
   const history = useHistory();
   const { action, setUserData } = props;
 
   const [showWarningModal, setShowWarningModal] = useState();
+  const [numberOfMinutes, setNumberOfMinutes] = useState();
```

Comment:
Can we give this an initial value? I noticed if I reload a page, we render the unset value briefly. Alternately, below you can flip the order of `setShowWarningModal(true);` and `setNumberOfMinutes(TIMEOUT_DISPLAY_TIME_IN_MINUTES);`.

Reply:
Hm, shouldn't refreshing the page remove the modal? Either way, I agree flipping the order makes sense!